src/utils/hints/accountHints.js

```diff
@@ -0,0 +1,14 @@
+export const accountHints = {
+  balance: 'Amount of AE currently owned by the account.',
+  value: 'Current fiat value of AE owned by the account.',
+  transactions: 'Amount of transactions sent from the account.',
+  aensNames: 'Amount of Names owned by the account.',
+  nonce: 'Value of the nonce for the account. Nonce serves as a one-time authenticator for transactions.',
+  apiLinks: 'Link to full details of the account.',
+  hash: 'Transaction hash of transaction sent by account.',
```

Comment:
It's not always sent, it could also be a transaction where the account is the receiver.

Reply:
fixed